File: Services/CategoryServices/DrivingSchoolApp.Services.CategoryServices.CCategoryAPI/DataAccess/Entities/Instructor.cs
namespace DrivingSchoolApp.Services.CategoryServices.CCategoryAPI.DataAccess.Entities
{
	public class Instructor : Base
	{
		public string Firstname { get; set; }
		public string Lastname { get; set; }
		public int Age { get; set; }
		public string LicenceId { get; set; }
		public string Photo { get; set; }
		public string StartedDay { get; set; }
		public int TotalWorkExperience { get; set; }
		public int TotalDriveExperience { get; set; }
		public int TruckDriveExperience { get; set; }
		public string InstructorAcademy { get; set; }
		public bool isBooked { get; set; }
		public int CategoryCId { get; set; }
		public virtual CategoryC CategoryC { get; set; }
		public IEnumerable<WorkExperience> WorkExperiencePerCompany { get; set; } = new List<WorkExperience>();
		public IEnumerable<InstructorCategoryLicence> CategoryLicences { get; set; } = new List<InstructorCategoryLicence>();
	}
}

File: Frontend/DrivingSchoolWeb/src/app/categories/category-c/models/instructor.ts
import {Base} from 'src/app/categories/shared/models/base';
import {WorkExperience} from 'src/app/categories/category-c/models/work-experience';
import {InstructorCategoryLicence} from 'src/app/categories/category-c/models/instructor-category-licence';

export interface Instructor extends Base{
  fullname: string,
  age: number,
  licenceId: string,
  photo: string | null,
  startedDay:string,
  totalWorkExperience:number,
  totalDriveExperience:number,
  truckDriveExperience:number,
  instructorAcademy:string,
  isBooked:boolean,
  categoryId:number,
  workExperiencePerCompany:WorkExperience[],
  categoryLicences:InstructorCategoryLicence[]
}
